test/spec/general.js

```diff
@@ -87,7 +87,7 @@ define(function(require) {
 
       setupVerifyIdTokenTest({
         title: 'verifies a valid idToken',
-        idToken: tokens.standardIdToken,
+        idToken: tokens.verifiableIdToken,
```

Comment:
Headless chrome will solve this problem on the cli, but until then, we have to run this manually before pushing.